### DISCUSSION TITLE
Example with postprocessing made from docker container

### DISCUSSION THREAD

**Discussion by mskyttner** [githubocto/flat-action#29] [Ideas & Feature Requests]
Thanks for the python example. Any possibility we could have a postprocessing example based on running not python but a command from a docker container?

**Comment by irealva** [comment]
Answered a potential [workaround here](https://github.com/githubocto/flat/discussions/5#discussioncomment-792637).

**Reply by mskyttner** [reply]
I seem to be able to run a command from inside a docker container (I tried rocker/tidyverse for an R script) with this approach: https://github.com/mskyttner/flat_data_bash_example/blob/main/.github/workflows/flat.yml#L13 (this repo has a fork with a slightly modified variant of https://github.com/aborruso/flat_data_bash_example)
